src/browser/declarativewebutils.cpp

```diff
@@ -121,8 +121,11 @@ void DeclarativeWebUtils::updateWebEngineSettings()
     SailfishOS::WebEngineSettings *webEngineSettings = SailfishOS::WebEngineSettings::instance();
     SailfishOS::WebEngine *webEngine = SailfishOS::WebEngine::instance();
 
+    MGConfItem userAgent("/apps/sailfish-browser/settings/user_agent");
+
     webEngineSettings->setPreference(QStringLiteral("general.useragent.updates.url"),
-                        QStringLiteral("https://browser.sailfishos.org/gecko/%APP_VERSION%/ua-update.json"));
+                        userAgent.value("https://browser.sailfishos.org/gecko/%APP_VERSION%/ua-updat e.json").value<QString>());
```

Comment:
Please, use `QStringLiteral`. Also you have an extra space character in your default string value (`ua-updat e.json`). For convenience you can use `toString()`.